libs/documentation/src/lib/components/autocomplete/demos/basic/autocomplete-basic.component.ts

```diff
@@ -39,7 +39,8 @@ export class AutocompleteBasic implements OnInit {
     this.settings.labelText = 'Autocomplete 1';
     this.settings.selectionMode = SelectionMode.SINGLE;
     this.settings.autocompletePlaceHolderText = 'eg: Level 1';
-    
+    this.settings.hideCloseIconPrimaryKeyValue = "1"
```

Comment:
Is this meant to be a toggle for displaying / hiding the close icon on autocomplete? I would expect it to be just a boolean no? True to hide otherwise just undefined for default behavior. What does the value of "1" represent here?

Reply:
I know, I implemented like that but they want to hide the Icon based on the option primary key value, they can't change the value to null or undefined.

Reply:
I think a better approach would be to go ahead and use a boolean - `hideCloseIcon` - that users can dynamically toggle. So as a user, I can listen for model change, check the model, and toggle the hideCloseIcon accordingly.